### ISSUE TITLE
tour: Not many insights about best practices or tips

### ISSUE THREAD

**Issue by CamiRev3** [golang/tour#1312]
Context: https://go.dev/tour/methods/18

There could be  multiple ways of doing this exercise.
My solution is: 

```
func (ip IPAddr) String() string {

	ipstr := fmt.Sprintf("%v", [4]byte(ip))

	ipstr = strings.ReplaceAll(strings.Trim(ipstr, "[]"), " ", ".")

	return ipstr
}

```
There should have any tip or best practice  explanation in terms of readability and efficiency.

**Comment by a3y3** [comment]
I agree with this - the practice exercises are good but they should also have solutions so that it's easier for students to know that they wrote code the right way. Just because it ran and passed all the tests doesn't mean the code was using best practices.
